SwiftCSVTests/CSVTests.swift

```diff
@@ -122,4 +122,14 @@ class CSVTests: XCTestCase {
             XCTAssert(error is CSVParseError)
         }
     }
+  
+    func testInit_ParseFileWithQuotesAndSpaces() {
```

Comment:
Since the implementation checks for `isWhitespace`, could you add similar-looking test cases for other whitespace characters, too?